lua/telescope/actions/state.lua

```diff
@@ -33,6 +33,7 @@ local select_to_edit_map = {
   horizontal = "new",
   vertical = "vnew",
   tab = "tabedit",
+  ["tab drop"] = "tab drop",
```

Comment:
uff did we forget to add drop in this table, in the previous PR?

Reply:
i wasn't involved in the previous PR, so i am not sure what this is referencing. i can definitely add it here though

Reply:
sorry thought you were the same person who added: https://github.com/nvim-telescope/telescope.nvim/commit/d793de0f12d874c463e81edabee741b802c1a37a

just because its also related to drop :laughing:

Reply:
no problem!